### PR TITLE
Created environment.yml and changed README.md

### LINKED ISSUE #9: Make requirements file work for ci, linux and windows

### PR DESCRIPTION
**Pull request by callistachang**
#14 
resolves #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,63 @@
...
+```
+# create virtual environment and install dependencies in environment.yml
+conda env create --file environment.yml
+# activate virtual environment
+conda activate poandy
+# when done
+conda deactivate
+```
+
+## Development
```
Comment (kaneishere) [inline README.md:20]: I think we should include the part where we need to set up a secret with our github account in order to be able to push to the repo.

**Review by kaneishere — COMMENTED** [review]
LGTM, please take a look at the comment I left

File: README.md (hunk 1 above)
Comment (wowweijie) [reply README.md:20]: for me, as long as readme has instructions for users

**Review by wowweijie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (callistachang) [reply README.md:20]: @kaneishere if CI removal is approved (in the other PR), there is no need to set up the secret anymore - else I'll include it in

**Review by kaneishere — APPROVED** [review]

**Comment by kaneishere** [comment]
Please squash your commits using this command
```
git reset --soft HEAD~6 && git commit -am "Created environment.yml and changed README.md"
git push -f
```

**Review by wowweijie — APPROVED** [review]

**Comment by callistachang** [comment]
> Please squash your commits using this command
> 
> ```
> git reset --soft HEAD~6 && git commit -am "Created environment.yml and changed README.md"
> git push -f
> ```

thanks @kaneishere ! done

**Pull request merged**